libnymea/plugin/plugin.pri

```diff
@@ -33,6 +33,14 @@ LIBS += -lnymea
 
 PLUGIN_PATH=/usr/lib/$$system('dpkg-architecture -q DEB_HOST_MULTIARCH')/nymea/plugins/
 
+QMAKE_CXXFLAGS *= -Werror -std=c++11 -g
+QMAKE_LFLAGS *= -std=c++11
+
+# Check if ccache is enabled
+ccache {
```

Comment:
hmm... this is odd...

I just have set this in my system and it uses ccache everywhere:

$ echo $PATH
/var/lib/ccache:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin

also our jenkins builders use that.

don't see why we'd want to use CONFIG+=ccache to only build the plugins with ccache...

Reply:
This file is for external projects, in this way you can choose to use ccache or not, depending on your env.